docs/UserGuide.md

```diff
@@ -203,6 +221,16 @@ Adds a person's contact detail to KonTActs.
 
 <box type="warning" icon=":fa-solid-circle-exclamation:" light>
 
+<md>**Name restrictions**</md>
+1. Names should only contain alphanumeric characters and whitespaces only. Special characters are not allowed.
+2. Names should be **unique**. Duplicate names cannot exist in the same database. Please refer to the [faq](#faq) on this issue.
+
```

Comment:
We could add a point mentioning that names have to be in English!

Reply:
I believe alphanumeric characters are all in English.

Reply:
Yup you are right as well, we can leave it as it is then!